src/changelog/2025/04/device-provisioning.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Remote Instance Provisioning
+description: "Bulk Registration now supports auto assigning devices to an application."
```

Comment:
"devices" don't exist anymore, it's "Remote instance"

Reply:
```suggestion
description: "Bulk Registration now supports auto assigning your Remote Instances to an Application."
```